test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -659,7 +660,7 @@ class cursor_bound_01 : public test {
                 auto srch_key = random_generator::instance().generate_random_string(
                   key_size, characters_type::ALPHABET);
 
-                int exact;
+                int exact = 0;
```

Comment:
np: Same here, I dont think we need to init since it is set in the `search_near` call but I guess Coverity being Coverity